cli/src/commands/extensions/api.rs

```diff
@@ -592,6 +589,12 @@ fn run_sandboxed(_process: Process) -> Result<ProcessOutput> {
     Err(anyhow!("Extension sandboxing is not supported on this platform"))
 }
 
+#[op]
+fn op_permissions(op_state: Rc<RefCell<OpState>>) -> permissions::Permissions {
+    let state = ExtensionState::from(op_state);
+    (*state.extension().permissions()).clone()
```

Comment:
Is this deref necessary here? Usually clone just automagically does this.

Reply:
`permissions()` is `Cow<_>`, so that would be `Cow::clone`. I don't love the deref myself. Does this look better?

```suggestion
    permission::Permission::clone(state.extension().permissions()))
```

Reply:
`Cow` has `.into_owned()` for exactly this purpose, doesn't it?